lib/test_prof/factory_prof/factory_girl_patch.rb

```diff
@@ -5,7 +5,7 @@ module FactoryProf
     # Wrap #run method with FactoryProf tracking
     module FactoryGirlPatch
       def run(strategy = @strategy)
-        FactoryProf.track(strategy, @name) { super }
+        TestProf::FactoryProf::FactoryBuilders::FactoryGirl.track(strategy, @name) { super }
```

Comment:
Can't we just use `FactoryBuilders::FactoryGirl` without `TestProf::FactoryProf::...`?